system/system_monitor/include/system_monitor/net_monitor/net_monitor.hpp

```diff
@@ -86,13 +87,29 @@ class NetMonitor : public rclcpp::Node
   void monitorTraffic(
     diagnostic_updater::DiagnosticStatusWrapper & stat);  // NOLINT(runtime/references)
 
+  /**
+   * @brief check IP packet reassembles failed
+   * @param [out] stat diagnostic message passed directly to diagnostic publish calls
+   * @note NOLINT syntax is needed since diagnostic_updater asks for a non-const reference
+   * to pass diagnostic message updated in this function to diagnostic publish calls.
+   */
+  void checkIpReasmFails(
```

Comment:
Please consider to use `checkReassemblesFailed`.

Reply:
@ito-san 
I changed the word as suggested.
24a196af79dda9c44c2781252396b25fd5df32f2